conf/locale/ru/LC_MESSAGES/django.po

```diff
@@ -16575,16 +16574,16 @@ msgid "You can then enroll in {course_name}."
 msgstr "Затем вы сможете записаться на курс «{course_name}»."
 
 #: lms/templates/emails/enroll_email_allowedsubject.txt
-msgid "You have been invited to register for {course_name}"
```

Comment:
consider to keep two translations - for original and for custom phrases. even when original is not used anymore it can appear in other places (modules, theme templates, etc, etc...)

Reply:
Got it, I'll update PR, thanks!

Reply:
fixed.